oarepo_vocabularies/ui/templates/semantic-ui/oarepo_vocabularies_ui/VocabulariesSidebar.jinja

```diff
@@ -1,11 +1,13 @@
 {#def record, extra_context #}
-<div style=" margin-top: 2em">
-<SidebarLink href={record.links.ui_links["edit"]} icon="edit">{{ _('Edit')}}</SidebarLink>
-{%if extra_context.vocabularyProps["hierarchical"] == True%}
-  <!-- TODO:(mesemus) we need ui_links['create'] passed by backend here -->
-  <SidebarLink icon="plus" href={ record['links']['vocabulary_html'] ~ '/_new?h-parent=' ~ record['id'] }>
-    {{_('New child item')}}
-  </SidebarLink>
+<div class="rel-mt-2">
+{%- if current_user and current_user.is_authenticated %}
+  <SidebarLink href={record.links.ui_links["edit"]} icon="edit">{{ _('Edit')}}</SidebarLink>
+  {%if extra_context.vocabularyProps["hierarchical"] == True%}
+    <!-- TODO:(mesemus) we need ui_links['create'] passed by backend here -->
```

Comment:
Could you please also look into this todo? Thanks :)

Reply:
@mirekys Hey, I've already checked it, but no url that goes to new is in the record's links

{'self': 'https://127.0.0.1:5000/api/vocabularies/institutions/amu', 'self_html': 'https://127.0.0.1:5000/vocabularies/institutions/amu', 'vocabulary': 'https://127.0.0.1:5000/api/vocabularies/institutions', 'vocabulary_html': 'https://127.0.0.1:5000/vocabularies/institutions', 'children': 'https://127.0.0.1:5000/api/vocabularies/institutions?h-parent=amu', 'children_html': 'https://127.0.0.1:5000/vocabularies/institutions?h-parent=amu', 'descendants': 'https://127.0.0.1:5000/api/vocabularies/institutions?h-ancestor=amu', 'descendants_html': 'https://127.0.0.1:5000/vocabularies/institutions?h-ancestor=amu', 'ui_links': {'self': 'https://127.0.0.1:5000/vocabularies/institutions/amu', 'edit': 'https://127.0.0.1:5000/vocabularies/institutions/amu/edit', 'search': 'https://127.0.0.1:5000/vocabularies/institutions/'}, 'export_path': '/vocabularies/institutions/amu/export', 'search_link': '/vocabularies/'}